### PR TITLE
Update abap-cloud-9aaaf65.md

### PR DESCRIPTION
**Pull request by iwonahahn**
- minor edits
- added link to SAP BTP ABAP environment documentation on SAP Help Portal

### CODE REVIEW

File: docs/design-a-multitenant-application-1ac614d.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ You have to classify database tables according to their content. There are the f
 
     Store data that is defined by the service provider and not specific for any tenant in a client-independent “S” table. Define the content in the respective development system and export it as TABU entries via a development transport request. The content is considered as code and imported like other development artifacts into subsequent systems such as the provider system.
 
-    Access to tables and all further ABAP Cloud Syntax is by default tenant aware.
+    Access to tables and all further ABAP Cloud Syntax is by default tenant-aware.
```
Comment (ekaterina-mitova) [inline docs/design-a-multitenant-application-1ac614d.md:46]: It should be tenant-aware syntax, or syntax that is tenant aware. I think it's correct in this case.

File: docs/tutorials-for-abap-cloud-fd87aaa.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The exercises are based on the Travel example of the [Flight Reference Scenario]
 
 ## Analytical Scenario
 
-The analytical scenario shows the end-to-end development of an analytical application. It consists of a business service exposing an analytical query and a report built with SAP Analytics Cloud. The analytical report can be used to analyze complex situations and freely interact with large amounts of data in real time.
+The analytical scenario shows the end-to-end development of an analytical application. It consists of a business service exposing an analytical query and a report built with SAP Analytics Cloud. The analytical report can be used to analyze complex situations and freely interact with large amounts of data in real-time.
```
Comment (ekaterina-mitova) [inline docs/tutorials-for-abap-cloud-fd87aaa.md:15]: This is the correct way to write it in this case.

**Review by ekaterina-mitova — APPROVED** [review]
Merging almost all the suggestions. Thank you so much for the valuable input!

**Comment by iwonahahn** [comment]
https://people.sap.com/iwona.hahn

**Pull request merged**